finetune/generate_qna_pairs.py

```diff
@@ -32,15 +32,29 @@
     default="TinyLlama/TinyLlama-1.1B-Chat-v0.4",
     help="path to chat model",
 )
-parser.add_argument(
```

Comment:
Reasoning for removing the sample size? Idea being of someone has thousands of chunks they may not want to generate questions for every single chunk

Reply:
I was thinking of allowing SE's to do more of a traditional train test split (80/20). I can keep the sample size argument and have the train test split depend on the sample size argument. What are your thoughts?

Reply:
sample size is added